blocks/tabs/tabs.css

```diff
@@ -1,46 +1,282 @@
-    padding: 8px 16px;
-    background-color: var(--light-color);
-    color: initial;
-    font-size: unset;
-    font-weight: bold;
-    line-height: unset;
-    text-align: initial;
-    text-overflow: unset;
-    overflow: unset;
-    white-space: unset;
-    transition: background-color 0.2s;
 }
 
-.tabs .tabs-list button[aria-selected="true"] {
-    border-bottom: 1px solid var(--background-color);
-    background-color: var(--background-color);
-    cursor: initial;
+.tabs-panel details summary span {
+    position: absolute;
+    width: 1rem;
+    height: 2px;
+    right: 1rem;
+    top: calc(50% - 2px);
+    background: var(--color-black);
+}
+
+.tabs-panel details summary span::after {
+    background: var(--color-black);
+    content: "";
+    position: absolute;
+    width: 2px;
+    height: 1rem;
+    top: -7px;
+    left: 7px;
+    transition: all .3s ease-in-out;
 }
 
-.tabs .tabs-list button[aria-selected="false"]:hover,
-.tabs .tabs-list button[aria-selected="false"]:focus {
+.tabs-panel details[open] summary span::after {
+    transform: rotate(90deg);
+}
+
+.tabs-panel details[open] summary::after {
+    transform: translateY(-50%) rotate(-45deg);
+}
+
+.tabs.vertical.block .tabs-panel > div:last-of-type > span::after {
+    background: var(--color-black);
+    content: "";
+    position: absolute;
+    width: 2px;
+    height: 1rem;
+    top: -7px;
+    left: 7px;
+    transition: all .3s ease-in-out;
+}
+
+.tabs.vertical.block .tabs-panel > div:last-of-type > span {
+    position: absolute;
+    width: 1rem;
+    height: 2px;
+    right: 1rem;
+    top: calc(50% - 2px);
+    background: var(--color-black);
+}
+
+.tabs .tabs-panel details {
+    width: 100%;
+    border-top: none;
+    border-bottom: 1px solid var(--color-light-silver);
+}
+  
+.tabs .tabs-panel:nth-of-type(2) details:first-of-type {
+    border-top: 1px solid var(--color-light-silver);
+}
+  
+
+.tabs-panel details[open] summary {
+    padding: .5rem 2.5rem .5rem .5rem;
+}
+
+.tabs-panel details summary:focus,
+.tabs-panel details summary:hover {
     background-color: var(--dark-color);
 }
 
-.tabs .tabs-panel {
-    margin-top: -1px;
-    padding: 0 16px;
-    border: 1px solid var(--dark-color);
-    overflow: auto;
+.tabs-panel details summary p {
+    color: var(--color-dark-gunmetal);
+    font-weight: var(--lato-bold);
+    font-size: var(--body-font-size-m);
+    line-height: 16px;
 }
 
-.tabs .tabs-panel[aria-hidden="true"] {
-    display: none;
+.tab-item-body {
+    max-height: 0;
+    overflow: hidden; 
+}
+
+@keyframes open-animation {
+    from {
+        padding: .125rem 1rem 1rem;
+        max-height: 0;
+    }
+
+    to {
+        padding: .125rem .5rem .5rem;
+        max-height: 5000vh;
+    }
+}
+
+@keyframes close-animation {
+    from {
+        padding: .125rem .5rem .5rem;
+        max-height: 5000vh;
+    }
+
+    to {
+        padding: .125rem 1rem 1rem;
+        max-height: 0;
+    }
+}
+
+.tab-item.open .tab-item-body {
```

Comment:
can we please add .block.tab before these
and can you merge 
.tab-item.close .tab-item-body with this